pkg/services/app/remove.go

```diff
@@ -99,7 +100,7 @@ func (a *App) getConfigUrlAndBranch(info *AppResourceInfo, token string) (string
 	if cloneURL == string(ConfigTypeUserRepo) {
 		cloneURL = info.Spec.URL
 	} else {
-		gitProvider, err := a.GitProviderFactory(token)
+		gitProvider, err := a.GitProviderFactory(token, gitproviders.GitProviderGitHub)
```

Comment:
Only supporting Github in `remove` for now?

Reply:
Yeah with my understanding of the ticket it was just for getting add to work in the app-config-url=none path.